.gitmodules

```diff
@@ -13,3 +13,6 @@
 	path = deps/nodesetLoader
 	url = https://github.com/open62541/nodesetLoader
 	branch = master
+[submodule "deps/libwebsockets"]
```

Comment:
Don't use a git submodule.
99% of users should install libwebsockets via a package manager.
And the remaining 1% know how to do it themselves.

Reply:
Remove the submodule. The users should manually install or use a distorted package.